keyboards/keyboardio/model01/info.json

```diff
@@ -97,70 +97,81 @@
     },
     "LAYOUT_thumb_row": {
```

Comment:
Yep, and I prefer this ordering over that of `LAYOUT`, but since there are user keymaps already using it I decided to leave it for now. There are a few other boards which do similar things eg. Ergodox, Kyria etc. so I might clean those all up later.